main_page.html

```diff
@@ -5,47 +5,105 @@
   <head>
 	<title>Main Page</title>
 	<link rel="stylesheet" type="text/css" href="css/upls_styles.css">
-	<script src="js/main_menu.js"></script>
+	<script defer src="js/main_menu.js"></script>
+	<script defer src="js/main_page.js"></script>
+	
   </head>
   
   <body>
 	<center>
+		
+		<div>
 		<img src="assets/placeholderlogo.png" alt="Logo for UPLP." class="main_logo">
+		<br>
+		
+		<button id="emailButton" class="large_button">Group Email</button>
+		<br>
+		
+		<button id="aggregateButton" class="large_button">Aggregate</button>
+		<br>
+		
+		<button id="addStudentButton" class="large_button">Add Student</button>	
+		<br>
+		
+		<button id="viewStudentButton" class="large_button">View Student</button>
+		<br>
+		
+		<button id="editStudentButton" class="large_button">Edit Student</button>
+		<br>
+		
+		<button id="addInteractionButton" class="large_button">Add Interaction</button>
+		<br>
+		
+		<button id="viewInteractionButton" class="large_button">View Interaction</button>
+		<br>
+		
+		<button id="editInteractionButton" class="large_button">Edit Interaction</button>
+		<br>
+		
+	    <button onclick="dropdown()" id="dropdown"  class="large_button admin">Admin Menu</button>
+		</div>
 		
-		<form action="add_student.html" class="">
-			<button type="submit" class="nav_button">Add Student</button>
-		</form>
-		
-		<form action="group_email.html" class="">
-			<button type="submit" class="nav_button">Group Email</button>
-		</form>
-		
-		<form action="aggregation.html" class="">
-			<button type="submit" class="nav_button">Aggregate</button>
-		</form>
-		
-		<form action="view_student.html" class="">
-			<button type="submit" class="nav_button">View Student</button>
-		</form>
-		
-	   <button onclick="dropdown()" id="dropdown"  class="admin_nav_button">Admin Menu</button>
-	   
-	   <div id="dropdown_items" class="dropdown_items">
-			 <br>
-			 <form action="batch_ingest.html" class="">
-				<button type="submit" class="admin_nav_button">Batch Ingest</button>
-			</form>
+	    <div id="dropdown_items" class="dropdown_items">
+			<br>
 			
-			<form action="edit_interaction_categories.html" class="">
-				<button type="submit" class="admin_nav_button">Interaction Categories</button>
-			</form>
+			<button id="ingestButton" class="large_button admin">Batch Ingest</button>
+			<br>
 			
-			<form action="edit_student_categories.html" >
-				<button type="submit" class="admin_nav_button">Student Categories</button>
-			</form>
+			<button id="intCategoryButton" class="large_button admin">Interaction Categories</button>
+			<br>
+			
+			<button id="studentCategoryButton" class="large_button admin">Student Categories</button>
 	
 		</div>
 	</center>
+	
+	<script>
```

Comment:
All of these can just be switched to a more concise approach in the html for each button. Just add onclick="yourFunctionName()".

Reply:
kept id-based approach, moved anonymous functions into js file (as discussed).